site-src/references/implementers-guide.md

```diff
@@ -0,0 +1,344 @@
+# Gateway API Implementer's Guide
+
+aka
+
+Everything you wanted to know about building a Gateway API implementation
+but were too afraid to ask.
+
```

Comment:
For those who don't speak english natively, the `aka` _might_ be a little confusing?

Not really a big deal I guess, but as a suggestion maybe you could just drop it and do this instead:

```suggestion
Everything you wanted to know about building a Gateway API implementation
but were too afraid to ask.

## Overview

```

Or something like that?